File: packages/sdk/src/bot/teamsBotSsoPromptTokenResponse.ts
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.
import { TokenResponse } from "botframework-schema";

/**
 * Token response provided by Teams Bot SSO prompt
 *
 * @beta
 */
export interface TeamsBotSsoPromptTokenResponse extends TokenResponse {
  /**
   * SSO token for user
   */
  ssoToken: string;

  /**
   * Expire time of SSO token
   */
  ssoTokenExpiration: string;
}

File: packages/dotnet-sdk/src/TeamsFx/Bot/TeamsBotSsoPromptTokenResponse.cs
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

using Microsoft.Bot.Schema;
namespace Microsoft.TeamsFx.Bot;

/// <summary>
/// Token response provided by Teams Bot SSO prompt
/// </summary>
public class TeamsBotSsoPromptTokenResponse: TokenResponse
{
    /// <summary>
    /// SSO token for user
    /// </summary>
    public string SsoToken { get; set; }

    /// <summary>
    /// Expiration time of SSO token
    /// </summary>
    public string SsoTokenExpiration { get; set; }
}
